omim2obo/main.py

```diff
@@ -207,6 +208,8 @@ def omim2obo(use_cache: bool = False):
             graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(exact_label, abbrev))))
         for label in other_labels:
             graph.add((omim_uri, oboInOwl.hasExactSynonym, Literal(label_cleaner.clean(label, abbrev))))
+        for included_label in cleaned_inc_labels:
+            graph.add((omim_uri, URIRef(INCLUDED_URI), Literal(label_cleaner.clean(included_label, abbrev))))
```

Comment:
Minor update: codestyle (line was too long)
- Made `INCLUDED_URI` and put it at the top of file

Reply:
Nice